src/Common/ValueObject/ValueObjectInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Common\ValueObject;
+
+interface ValueObjectInterface
```

Comment:
this interface should not exist and should be replaced by `Stringable`, which exists since PHP 8.0, and is available from `symfony/polyfill-php80`
[See here](https://github.com/symfony/polyfill-php80/blob/1.x/Resources/stubs/Stringable.php)